tests/functional/test_hash_address.py

```diff
@@ -0,0 +1,109 @@
+import pytest
+
+from ape.api.address import BaseAddress
+from ape.types.address import AddressType
+
+
+class TestAddress(BaseAddress):
```

Comment:
We have some built in account objects we can probably test with instead (see `accounts` fixture)

Reply:
A class in pytest land is different

Reply:
Oh, it is not actually being used this way... Ya, I agree with @fubuloubu then.